cmake/flash/CMakeLists.txt

```diff
@@ -0,0 +1,77 @@
+get_property(ENV_VARS GLOBAL PROPERTY FLASH_SCRIPT_ENV_VARS)
+list(APPEND ENV_VARS
+  O=${PROJECT_BINARY_DIR}
+  KERNEL_ELF_NAME=${KERNEL_ELF_NAME}
+  KERNEL_HEX_NAME=${KERNEL_HEX_NAME}
+  KERNEL_BIN_NAME=${KERNEL_BIN_NAME}
```

Comment:
need to export ARCH and BOARD_NAME as well, needed for openocd

Reply:
@nashif : Fixed.